services/execution/service.go

```diff
@@ -129,7 +130,7 @@ func (s *Service) Create(ctx context.Context, r *api.CreateRequest) (*api.Create
 	}
 	c, err := runtime.Create(ctx, r.ContainerID, opts)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "runtime create failed")
```

Comment:
I don't think it's doable at this level, that code would need to come from the lower level